contracts/vaults/STETHVault/RibbonThetaSTETHVault.sol

```diff
@@ -269,16 +271,45 @@ contract RibbonThetaSTETHVault is RibbonVault, RibbonThetaSTETHVaultStorage {
 
         emit InstantWithdraw(msg.sender, amount, currentRound);
 
+        IERC20 steth = IERC20(collateralToken.stETH());
+        uint256 startStethBalance = steth.balanceOf(address(this));
+        bool enoughSTETH = true;
```

Comment:
enoughSTETH = (steth + wsteth) balance > withdraw amount. imagine someone deposits 1 eth and is only person in vault, and then wants to withdraw 1steth. this won't work so we will have to give them eth (since there is no steth balance or wsteth balance).